components/i18n.tsx

```diff
@@ -33,11 +77,11 @@ const i18n: { [key: string]: i18nLang } = {
       On Discord, more cool: Gauthier#0875 (or on <a href="https://discord.gg/vCjEufP" target="_blank">my server</a>)
     </>,
     'About page github': () => <>
-      My GitHub, to check my projects: <a href="https://github.com/gauthier-th">https://github.com/gauthier-th</a>
+      My GitHub, check my projects: <a href="https://github.com/gauthier-th">https://github.com/gauthier-th</a>
```

Comment:
I think the original is ok

Reply:
No, _to check_ is in German, _überprüfen_! So would be:

> Mein GitHub, um meine Projekte zu überprüfen

What sounds stupid! You don't check (or review) projects in this case, you look at them! Is a difference

Reply:
_My GitHub, check my projects_ can be translated in german as

> Mein GitHub, checke meine Projekte

which sounds much better and more suitable

Reply:
Ok